src/ui/components/Transfer.js

```diff
@@ -79,6 +87,7 @@ export const Transfer = ({
   const [receiver, setReceiver] = useState<Account | null>(null);
   const [amount, setAmount] = useState<string>("");
   const [memo, setMemo] = useState<string>("");
+  const [isSavedToLedger, setIsSavedToLedger] = useState<boolean>(false);
```

Comment:
This name is a little confusing, because I expect it to be `false` in dirty, but it's actually the opposite. A clearer name would be `hasUnsavedChanges`